front/src/modules/eslint-plugin-twenty/rules/no-hardcoded-colors.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
I don't think this should leave within a src/module. Right now, modules are application level files (business logic, ui...), they are not development level files. 
Also I don't think we should create a plugin at this point. Would a eslint-rules at root be enough for now?

Reply:
I see your point. Putting it directly at the routes doesn't seem very clean but maybe we can create a "modules" or "packages" folder at the root?  What do you think? But I'm afraid it will create noise and make it harder to understand the repo for someone new (front VS server is very clean, packages add a confusion)

I didn't manage to create a standalone rule without creating a plugin. I feel like creating a plugin is the right maybe to go because it allows anyone to create rules easily in the future